phpunit/functional/LockedfieldTest.php

```diff
@@ -924,4 +924,48 @@ public function testReplaceLockedField()
         $this->assertTrue($computer->getFromDB($cid));
         $this->assertEquals($computer->fields['locations_id'], 0);
     }
+
+    public function testCheckAllInventoryLockableObjects()
+    {
+        $this->login('glpi', 'glpi');
+
+        /** @var \DBmysql $DB */
+        global $DB;
+
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+        foreach ($CFG_GLPI['inventory_lockable_objects'] as $itemtype) {
+            $this->assertTrue($DB->fieldExists($itemtype::getTable(), 'is_dynamic'));
+        }
+
+
+        // Excluded type with is_dynamic field but not in inventory_lockable_objects
+        $excluded = [
+            'UserEmail',
+            'Group_User',
+            'Profile_User',
+        ];
+
+        $global_inventory_type = array_merge(
+            $CFG_GLPI['inventory_lockable_objects'],
+            $CFG_GLPI['inventory_types']
+        );
+
+        $tables = $DB->listTables();
+        foreach ($tables as $table_data) {
+            $table = $table_data['TABLE_NAME'];
+
+            if ($DB->fieldExists($table, 'is_dynamic')) {
+                $itemtype = getItemTypeForTable($table);
+                if (in_array($itemtype, $excluded)) {
+                    continue;
+                }
+
+                $this->assertContains(
+                    $itemtype,
+                    $global_inventory_type,
+                );
```

Comment:
```suggestion
                $this->assertContains(
                    $itemtype,
                    $global_inventory_type,
                    "Add a clear message here in case of validation failure."
                );
```
can you add message here